azure-proto-core/SubscriptionContainerOperations.cs

```diff
@@ -0,0 +1,46 @@
+using Azure;
+using Azure.ResourceManager.Resources;
+using Azure.ResourceManager.Resources.Models;
+using azure_proto_core.Adapters;
+using System;
+using System.Threading;
+
+namespace azure_proto_core
+{
+    /// <summary>
+    /// Subscription Container Operations
+    /// </summary>
+    public class SubscriptionContainerOperations : OperationsBase
+    {
+        public static readonly string AzureResourceType = "Microsoft.Resources/subscriptions";
+
+        public SubscriptionContainerOperations(ArmClientContext context, string defaultSubscription) : base(context, $"/subscriptions/{defaultSubscription}") { }
```

Comment:
typically the container takes the resource/id of its parent.  The parent of a subscription is loosely a tenant, but currently its armClient so I wonder if the only parameter to this class should be ArmClientContext?